webserver/templates/navbar.html

```diff
@@ -28,7 +28,7 @@
       {# Not showing user stuff on error pages. If attempt to load user info fails there, page will not render. #}
       {% if not error %}
         <ul class="nav navbar-nav navbar-right">
-          {% if not current_user or current_user.is_anonymous() %}
+          {% if not current_user or current_user.is_anonymous %}
```

Comment:
I'm guessing this is because `is_anonymous` and a bunch of other values have been converted into class properties in one of the latest versions of Flask-Login?